content/en/project/Ridani_Chi_separation_project/index.md

```diff
@@ -0,0 +1,186 @@
+## About Me
```

Comment:
Please add the header for the project, including metadata. You can adapt the following template: 

```
---
type: "project" # DON'T TOUCH THIS ! :)
date: "2020-05-16" # Date you first upload your project.
# Title of your project (we like creative title)
title: "This is an example project page which serves as a template"

# List the names of the collaborators within the [ ]. If alone, simple put your name within []
names: [Samuel Guay, Pierre Bellec]

# Your project GitHub repository URL
github_repo: https://github.com/brainhack-school2023/project_template

# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
website:

# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/brainhack-school2020/project_template), click `manage topics`.
# Please only lowercase letters
tags: [project, github, markdown, brainhack]
```

Reply:
Hi @pbellec , sorry about that, I already did it before, I don't know if you can check the history of the commits, but I think I deleted it by accident when I was editing. So, thank you so much for the feedback!!